puzzle/cli/cli.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+import os
+import logging
+
+import click
+
+from codecs import open
+from puzzle.log import configure_stream, LEVELS
+
+from puzzle.settings import BaseConfig
+from puzzle.plugins import SqlStore, VcfPlugin
+
+from . import (root, verbose)
+
+logger = logging.getLogger(__name__)
+
+
+@click.group()
+@root
```

Comment:
Since `root` is an option in all the subcommands I think we should avoid it here to avoid confusion about what makes this different from the rest, agree?